diff --git a/cmake/FindQtWaylandScanner.cmake b/cmake/FindQtWaylandScanner.cmake
index 0e97535..fa26738 100644
--- a/cmake/FindQtWaylandScanner.cmake
+++ b/cmake/FindQtWaylandScanner.cmake
@@ -90,7 +90,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/ECMFindModuleHelpers.cmake)
 ecm_find_package_version_check(QtWaylandScanner)
 
 # Find qtwaylandscanner
-find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner HINTS /usr/lib64/qt5/bin/)
+find_program(QtWaylandScanner_EXECUTABLE NAMES qtwaylandscanner HINTS /usr/lib/qt5/bin/)
 
 include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(QtWaylandScanner
